From 0b8573fef7f281b807f86844109ee6a9da3bf168 Mon Sep 17 00:00:00 2001
From: Huawei <@huawei.com>
Date: Tue, 26 Nov 2024 15:36:00 +0800
Subject: [PATCH 2/2] net/meson: add support for hinic3 DPDK pmd driver

Add support for hinic3 DPDK pmd driver to driver/net/meson.build.

Signed-off-by: Huawei <@huawei.com>
---
 drivers/net/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/meson.build b/drivers/net/meson.build
index 2355d1cde8..f568f45854 100644
--- a/drivers/net/meson.build
+++ b/drivers/net/meson.build
@@ -24,6 +24,7 @@ drivers = [
         'failsafe',
         'fm10k',
         'hinic',
+        'hinic3',
         'hns3',
         'i40e',
         'iavf',
-- 
2.46.0.windows.1

